packages/builders/scripts/vats/terminate-governed-instance.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * @file Terminate price-feed governor instances such as mainnet v110.
+ * Functions as both an off-chain builder and an on-chain core-eval.
+ */
+
+/// <reference types="@agoric/vats/src/core/types-ambient"/>
+
+import { E } from '@endo/far';
+
+const SELF = '@agoric/builders/scripts/vats/terminate-governed-instance.js';
+const USAGE = `Usage: agoric run /path/to/terminate-governed-instance.js \\
+  <$governorInstanceHandleBoardID:$instanceKitLabel>...`;
+
+const repr = val =>
+  typeof val === 'string' || (typeof val === 'object' && val !== null)
+    ? JSON.stringify(val)
+    : String(val);
+const defaultMakeError = (strings, ...subs) =>
+  Error(
+    strings.map((s, i) => `${i === 0 ? '' : repr(subs[i - 1])}${s}`).join(''),
+  );
+const makeUsageError = (strings, ...subs) => {
+  const err = defaultMakeError(strings, ...subs);
+  console.error(err.message);
+  console.error(USAGE);
+  return err;
+};
+
+const rtarget = /^(?<boardID>board[0-9]+):(?<instanceKitLabel>.+)$/;
+/**
+ * @param {string[]} args
+ * @param {(strings: TemplateStringsArray | string[], ...subs: unknown[]) => Error} [makeError]
+ * @returns {Array<{boardID: string, instanceKitLabel: string}>}
+ */
+const parseTargets = (args = [], makeError = defaultMakeError) => {
+  if (!Array.isArray(args)) throw makeError`invalid targets: ${args}`;
+  /** @type {Array<{boardID: string, instanceKitLabel: string}>} */
+  const targets = [];
+  const badTargets = [];
+  for (const arg of args) {
+    const m = typeof arg === 'string' && arg.match(rtarget);
+    if (!m) {
+      badTargets.push(arg);
+    } else {
+      // @ts-expect-error cast
+      targets.push(m.groups);
+    }
```

Comment:
Indent-reduction suggestion:
```suggestion
    if (!m) {
      badTargets.push(arg);
      continue;
    }
    // @ts-expect-error cast
    targets.push(m.groups);
```